app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenControllerTest.kt

```diff
@@ -535,12 +509,13 @@ class PatientEntryScreenControllerTest {
     inOrder.verify(ui).scrollToFirstFieldWithError()
   }
 
-  @Test // TODO: Migrate to Mobius
+  @Test
   fun `when the ongoing entry has an identifier it must be retained when accepting input`() {
     val identifier = Identifier(value = "id", type = BpPassport)
     whenever(patientRepository.ongoingEntry()).thenReturn(Single.just(OngoingNewPatientEntry(identifier = identifier)))
     whenever(patientRepository.saveOngoingEntry(any())).thenReturn(Completable.complete())
     whenever(patientRegisteredCount.get()).thenReturn(0)
+    screenCreatedForMobius()
```

Comment:
This can just renamed to `screenCreated` now, and the older method can be removed.